HISTORY.md

```diff
@@ -4,6 +4,7 @@
 
 - Removed `log.tqdm`
 - Added binning functions to `pelutils.ds.plot`
+- Added `pelutils.ds.distributions` with `scipy` distributions that use same notation as Jim Pitman's "Probability"
```

Comment:
Please shorten commit message. Eventually by writing Pitman instead of Jim Pitman

Reply:
What should I do in the meantime?